tools/autograd/derivatives.yaml

```diff
@@ -631,10 +631,12 @@
 
 - name: fmod.Scalar(Tensor self, Scalar other) -> Tensor
   self: grad
+  result: self_t
 
 - name: fmod.Tensor(Tensor self, Tensor other) -> Tensor
   self: grad
   other: 'not_implemented("fmod: other")'
+  result: self_t.broadcast_to(result.sizes())
```

Comment:
That is not correct? or you should have a check that ensures that `other` is always zeros.
Otherwise, we should implement both the forward and backward formulas related to other.

Reply:
Whoops, I think I need to study this operator more. For this PR, I will just revert this change since I realized @Lezcano already has PR out for this.

Reply:
For reference, I have one that does this one and it's cousin `torch.reminder`.